bonsai/examples/governance/methods/README.md

```diff
@@ -0,0 +1,34 @@
+## zkVM Methods
+
+This directory contains the [zkVM] portion of your [Bonsai] application. 
+This is where you will define one or more [guest programs] to act as a coprocessor to your [on-chain logic].
+
+> In typical use cases, the only code in this directory that you will need to edit is inside [`guest/src/bin`]. 
+
+### Writing Guest Code
+To learn to write code for the zkVM, we recommend [Guest Code 101]. 
+Note that you don't need to write host code because the Bonsai Ethereum Relay acts as the zkVM host.
+
+Examples of what you can do in the guest can be found in the [RISC Zero examples].
+
+### From Guest Code to Binary File
+
+Code in the `methods/guest` directory will be compiled into one or more [RISC-V] binaries. 
+Build configuration for the methods is included in `methods/build.rs`.
+
+Each will have a corresponding image ID, which is a hash identifying the program.
+
+### Uploading Binary to Bonsai
+When [deploying] your application, you will upload your binary to Bonsai where the guest will run when requested.
+The image ID will be included in the deployment of the smart contracts to reference your guest program living in Bonsai.
```

Comment:
Do users need to upload their own binary of this to Bonsai? Or will we have one already uploaded? I'm unclear (in particular, I don't know whether governing different things requires different ImageIDs or just different inputs). In the former case we should rewrite this to make it clear we're talking about specifically for the governance application, not Bonsai apps in general, and to make clear under what conditions a new deployment is needed. In the latter case, this section should be dropped and we should write a section explaining how to use the existing deployment.